Themes/Gray Matter (Dark).theme.json

```diff
@@ -0,0 +1,128 @@
+{
+    "name": "Writer",
+    "style": "Dark",
+    "author": [
```

Comment:
You should be able to add any field that's not yet taken and provide a link to your VSCode theme, if you want, like `"basedOn" : "https://github.com/philipbelesky/gray-matter"` or something. Either at the project root, or as a note in the author subcategory, or whatever suits you best :)